addon/src/build-waiter.ts

```diff
@@ -128,7 +128,7 @@ class NoopTestWaiter implements TestWaiter {
 /**
  * Builds and returns a test waiter. The type of the
  * returned waiter is dependent on whether the app or
- * addon is in `isDevopingApp()` mode or not.
```

Comment:
I may like to Dev Oping, more like Dev Oopsing lol

Reply:
lol